libraries/AP_Notify/Display.cpp

```diff
@@ -488,11 +488,24 @@ void Display::update_mode(uint8_t r)
 	}
 }
 
+void Display::update_text_empty(uint8_t r)
+{
+    char msg [DISPLAY_MESSAGE_SIZE];
+    memset(msg, ' ', sizeof(msg));
+    draw_text(COLUMN(0), ROW(0), msg);
```

Comment:
My bad.  Can someone tell me how msg gets nul-terminated in the main update_text function, 'though?